portfolio/src/main/webapp/script.js

```diff
@@ -58,9 +58,19 @@ function playRandomSportsScene() {
   videoContainer.appendChild(iframe);
 }
 
-async function addServletContentToDOM() {
+async function loadCommentsToDOM() {
   const response = await fetch('/data');
   const content = await response.json();
-  const servletContainer = document.getElementById('servlet-container');
-  servletContainer.innerHTML = content;
+  const servletContainer = document.getElementById('comment-section');
+
+  for(i = 0; i < content.length; i++) {
```

Comment:
`let i = 0`?

Reply:
I just saw that and went ahead and resolved the issue. I'm a little bit confused why it worked before though. I think I overlooked the error because the functionality was still there.